Description: Retain commented APT partner pocket in sources.list template
Author: chad.smith@canonical.com
Origin: backport
Forwarded: not-needed
Last-Update: 2022-02-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/templates/sources.list.ubuntu.tmpl
+++ b/templates/sources.list.ubuntu.tmpl
@@ -43,6 +43,13 @@ deb {{mirror}} {{codename}}-updates mult
 deb {{mirror}} {{codename}}-backports main restricted universe multiverse
 # deb-src {{mirror}} {{codename}}-backports main restricted universe multiverse
 
+## Uncomment the following two lines to add software from Canonical's
+## 'partner' repository.
+## This software is not part of Ubuntu, but is offered by Canonical and the
+## respective vendors as a service to Ubuntu users.
+# deb http://archive.canonical.com/ubuntu {{codename}} partner
+# deb-src http://archive.canonical.com/ubuntu {{codename}} partner
+
 deb {{security}} {{codename}}-security main restricted
 # deb-src {{security}} {{codename}}-security main restricted
 deb {{security}} {{codename}}-security universe
